infra/firebase.tf

```diff
@@ -20,3 +20,11 @@ resource "google_firebase_project" "default" {
 
   depends_on = [google_project_service.enabled]
 }
+
+resource "google_firebase_hosting_site" "client" {
+  provider = google-beta
+  project  = var.project_id
+  site_id  = var.random_suffix ? "${var.project_id}-${random_id.suffix.hex}" : var.project_id
```

Comment:
potential blocker: Are we missing a parallel change in [infra/outputs](https://github.com/GoogleCloudPlatform/terraform-dynamic-python-webapp/tree/b260df6321e9e2f5e1ac32fdeef00abb6f2ce1e2/infra) for the two locations in that file that output the firebase site URL?

Reply:
oh yes you're very much right.